src/main/fc/settings.yaml

```diff
@@ -4092,3 +4083,51 @@ groups:
         field: osd_joystick_enter
         min: 0
         max: 100
+
+  - name: PG_FW_AUTOLAND_CONFIG
```

Comment:
@DzikuVx this is changing and adding new configuration items and will not allow flash without erase from 7.0 to 7.1.

Is this ok, or should it be pushed to 8.0 instead?

Reply:
This is not a problem actually and for this full chip erase is not required. It's a new PG and will be automatically added during 1st boot. The problem is different: some settings were removed / renamed. On the other hand, removing a setting completely is not a problem. And the one that was renamed keeps the same value.

Reply:
if that is not an issue, I have no problems with new features getting released sooner